spec/features/user_can_see_all_books_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+describe "User logs in" do
+  scenario "sees book title" do
+    user = User.create(name: "Sally")
+    book = Book.create(title: "Harry Potter")
+    review = book.reviews.create(body: "love this story", rating: 5)
```

Comment:
You would have wanted to pass your user into this review.create. It likely won't save a review without a user since a Review belongs_to a User.